<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>320</y>
     <width>351</width>
     <height>311</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Temperature</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Humidity</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Time</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="button2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>281</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Get 10 historic points</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>640</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0); font-size: 18pt;
background-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>Alarms:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>640</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 18pt;
color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string>Temperature</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>640</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 18pt; 
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Humidity</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>680</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>-20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>40</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>50</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>60</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>70</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>80</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>90</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>100</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>680</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>40</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>50</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>60</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>70</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>80</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>90</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>670</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Get current weather</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>401</width>
     <height>71</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Temperature</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Humidity</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Time</string>
    </property>
   </column>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Display in fahrenheit/celsius</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
